go.mod

```diff
@@ -1,14 +1,14 @@
 module k8s.io/cloud-provider-vsphere
 
-go 1.16
+go 1.17
```

Comment:
Could you set it to 1.16 still. Since I saw k8s is still referring to 1.16
https://github.com/kubernetes/kubernetes/blob/v1.23.1/go.mod#L9

Reply:
Unfortunately I can't, because some libraries explicitly require golang 1.17: https://github.com/kubernetes-sigs/controller-runtime/blob/master/go.mod#L3

Without this bump the code wouldn't compile.